organizer-contacts.md

```diff
@@ -0,0 +1,9 @@
+# Organizer Contacts
+### Name / e-mail / Twitter handle / Github / dewp.slack.com
+
+* André Brückner / andre.brueckner@posteo.de / [tichypress] (https://twitter.com/tichypress) / [tichypress] (https://github.com/tichypress) / tichypress
+* Alexander (Alex) Goller / alpipego@gmail.com / [alpipego] (https://twitter.com/alpipego) / [alpipego] (https://github.com/alpipego) / alpipego
+* David Stingl / mail@david-stingl.de / [berlindave] (https://twitter.com/berlindave) / [berlindave] (https://github.com/berlindave) / berlindave
+* Stefan Fröhlich / stefan@froehlich.it / ? / ? / –
+* Sven Brier / sb@netzgrad.org / [svenbrier] (https://twitter.com/svenbrier) / [svenbrier] (https://github.com/svenbrier) / svenbrier
+* Sven Hofmann / info@hofmannsven.com / [hofmannsven] (https://twitter.com/hofmannsven) / [hofmannsven] (https://github.com/hofmannsven) / hofmannsven
```

Comment:
Better remove the space between handle and link in [Markdown files](https://guides.github.com/features/mastering-markdown/#examples).

Reply:
Than you for the hint! Unfortunately it worked with the space in my Markdown editor.